app/database/group_attempt_store_compute_all_group_attempts.go

```diff
@@ -102,7 +102,7 @@ func (s *GroupAttemptStore) ComputeAllGroupAttempts() (err error) {
 							MAX(IF(items_items.category = 'Validation', aggregated_children_attempts.validated_at, NULL))
 								AS max_validated_at_categories,
 							SUM(IFNULL(aggregated_children_attempts.score, 0) * items_items.score_weight) /
-								COALESCE(NULLIF(IFNULL(SUM(items_items.score_weight), 0), 0), 1) AS average_score
+								COALESCE(NULLIF(SUM(items_items.score_weight), 0), 1) AS average_score
```

Comment:
usually you use `IFNULL` for doing `COALESCE`

Reply:
IFNULL and NULLIF are not the same thing

Reply:
I didn't say that. I say that `IFNULL` and `COALESCE` are the same thing. And at other places, you prefer using `IFNULL` for that use, so it is not consistent with what you do usually.

Reply:
Do you prefer 
```
IFNULL(NULLIF(SUM(items_items.score_weight), 0), 1)
```
to
```
COALESCE(NULLIF(SUM(items_items.score_weight), 0), 1)
```
?

Reply:
It is not a matter of preference, I just say that you usually use `IFNULL` for this usage. 
(and you did `IFNULL(NULLIF(` in the past already).
You decide.

Reply:
In this particulare case, for me, 'COALESCE(NULLIF' is more readable than 'IFNULL(NULLIF'